hw-1/task2.js

```diff
@@ -0,0 +1,16 @@
+import csv from 'csvtojson';
+import * as path from 'path';
+
+console.log(csv.csv);
+
+const CSV_FILE_PATH = path.join(__dirname, './input.csv');
+const TEXT_FILE_PATH = path.join(__dirname, './output.txt');
+
+const readStream = require('fs').createReadStream(CSV_FILE_PATH);
+const writeStream = require('fs').createWriteStream(TEXT_FILE_PATH);
+
+readStream
+    .pipe(csv({
```

Comment:
You need to use 2 spaces, or 4 spaces everywhere, not different amount in different files

Reply:
Done